apps/ejabberd/src/mongoose_admin.erl

```diff
@@ -181,9 +166,8 @@ register(User, Host, Password) ->
             throw({error, String})
     end.
 
-unregister(Jid) ->
-    J = jid:from_binary(Jid),
-    ejabberd_auth:remove_user(J#jid.luser, J#jid.lserver),
+unregister(Host, User) ->
+    ejabberd_auth:remove_user(User, Host),
     <<"">>.
```

Comment:
Perhaps we could return something more meaningful than `<<"">>` (even it doesn't get into an HTTP response).